python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -1203,8 +1203,8 @@ def test_to_from_arrow_nulls(data_type):
     # We have 64B padded buffers for nulls whereas Arrow returns a minimal
```

Comment:
This comment is surprising. Arrow spec requires 64B padded buffers for masks, so why is it returning minimal bytes?

Reply:
I think this is a PyArrow implementation detail that we could likely resolve by using their Cython / C++ APIs, but is out of scope for this PR.